### PR TITLE
AYS-447 | Correcting the Reference Number on the 'Emergency Evacuation Application Details' Screen

### PR DESCRIPTION
**Pull request by gafur-demirci**
### Checklist

Before submitting your pull request, ensure the following:

- [ ] **Change Type:**
- [ ] Bug fix
- [ ] Feature addition
- [ ] Documentation update
- [ ] Performance improvement
- [x] Other (refactor)

- [x] **Summary:**

  - The Reference Number on the Emergency Evacuation Application Details page is formatted using the Reference Number function..

- [x] **Title and Branch Naming Conventions:**

  - The pull request title follows the
    standard: [Pull Request Naming Conventions](CONTRIBUTING.md#pull-request-naming-conventions).
  - The branch name follows one of the conventions outlined in
    the [Branch Naming Conventions](CONTRIBUTING.md#branch-naming-conventions).

- [x] **Local Testing:**

  - I have tested my changes locally on my browser, and they are working as expected.

- [x] **Code Quality:**

  - The code is formatted according to the project's coding guidelines and style.
  - The code has been reviewed to ensure its quality.
  - The code does not contain any issues flagged by ESLint.

- [x] **Documentation:**

  - Necessary documentation has been added or existing documentation has been updated.

- [x] **Reviewers and Assignees:**

  - Default reviewers have been assigned to this pull request.
  - Assignees have been added if necessary.

- [x] **Related Issue:**
  - [AYS-447](https://afetyonetimsistemi.atlassian.net/browse/AYS-447)

- [x] **Labels and Associations:**
  - No specific actions are required in the Labels and Associations section for this pull request.

[AYS-447]: https://afetyonetimsistemi.atlassian.net/browse/AYS-447?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/app/(private)/emergency-evacuation-applications/[id]/page.tsx [hunk 1]
```diff
@@ -90,8 +91,12 @@ const Page = ({
                               {...field}
                               disabled
                               defaultValue={
-                                emergencyEvacuationApplicationDetails.referenceNumber ??
-                                ''
+                                emergencyEvacuationApplicationDetails?.referenceNumber ?
+                                  formatReferenceNumber(
+                                    emergencyEvacuationApplicationDetails.referenceNumber
+                                  )
+                                  :
```
Comment (sanshigo345) [inline src/app/(private)/emergency-evacuation-applications/[id]/page.tsx:98]: "emergencyEvacuationApplicationDetails?.referenceNumber" burada ? işaretine gerek var mı? sanırım yok diğer yerlerde de böyle bi durum varsa değiştirebiliriz

**Review by sanshigo345 — CHANGES_REQUESTED** [review]
eslint hatalarını düzeltmek lazım

File: src/app/(private)/emergency-evacuation-applications/[id]/page.tsx (hunk 1 above)
Comment (gafur-demirci) [reply src/app/(private)/emergency-evacuation-applications/[id]/page.tsx:98]: Backend olarak kontrol etmedim, ama UI'ı güvenceye almak için eklemiştim.

Comment (sanshigo345) [reply src/app/(private)/emergency-evacuation-applications/[id]/page.tsx:98]: {!isLoading && !error && emergencyEvacuationApplicationDetails && (

yukarda böyle bir check olduğu için gerek yok diye düşündüm, sen de gerek olmadığını düşünüyosan buradan ve diğer yerlerden silebiliriz 👍

Comment (gafur-demirci) [reply src/app/(private)/emergency-evacuation-applications/[id]/page.tsx]: Gerekli düzenlemeler yapılıp, kontrol edildi. Teşekkürler 🙏🏼

**Review by sanshigo345 — APPROVED** [review]

**Pull request merged**